packages/block-library/src/navigation/index.php

```diff
@@ -85,6 +85,36 @@ function block_core_navigation_build_css_font_sizes( $attributes ) {
 	return $font_sizes;
 }
 
+/**
+ * Renders a Navigation Block derived from data from the theme_location assigned
+ * via the block attribute 'location'.
+ *
+ * If the theme doesn't explicity support 'block-nav-menus' or no location was provided
+ * as a block attribute then an empty string is returned.
+ *
+ * @param  array $attributes Navigation block attributes.
+ * @return string HTML markup of a generated Navigation Block.
+ */
+function get_classic_navigation_elements( $attributes ) {
```

Comment:
We could rename `get_classic_navigation_elements` to `render_classic_location_menu` because that is what the function does, and the getter aspect is not correct. I would also change the signature to include `$location` as a parameter since it's a bit opaque otherwise => `render_classic_navigation( $location, $attributes ) : mixed {}`; I'd make it return `false` for an empty location.